catalogmanagementv1/catalog_management_v1.go

```diff
@@ -15,7 +15,7 @@
  */
 
 /*
- * IBM OpenAPI SDK Code Generator Version: 3.70.0-7df966bf-20230419-195904
+ * IBM OpenAPI SDK Code Generator Version: 3.68.2-ac7def68-20230310-195410
```

Comment:
Why did you use an older version of the SDK generator as compared to the last time this service was generated?